homeassistant/components/ezviz/camera.py

```diff
@@ -109,11 +110,23 @@ async def async_setup_entry(hass, entry, async_add_entities):
             )
 
         else:
+
+            await hass.config_entries.flow.async_init(
```

Comment:
Create a task instead of awaiting. Use `hass.async_create_task`.

Reply:
Done.